exporter/collector/integrationtest/testcase.go

```diff
@@ -48,6 +50,92 @@ var (
 	}
 )
 
+type LogsTestCase struct {
```

Comment:
Is this exactly the same as MetricsTestCase?  I'm all for refactoring later, but for identical structs, it is probably worth unifying them here.

Reply:
done, they have all similar functions that I just renamed for now but can probably be refactored to a common interface later